### PR TITLE
I18n change vocabularies

### PR DESCRIPTION
**Pull request by frodrigo**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -107,17 +107,17 @@ en:
...
-        one: Store
-        other: Stores
-      tag: Tag
+        one: Facility
+        other: Facilities
+      tag: Category
       tags:
-        one: tag
-        other: tags
+        one: Category
```
Comment (braktar) [inline config/locales/en.yml:119]: Some words were starting with a small case. Did you've identified where it was involved ?

File: config/locales/en.yml [hunk 2]
```diff
@@ -546,8 +546,8 @@ en:
         time_window_start_2_time_window_end_2_days: D+
         priority: Priority
         multi_visit: Multi
-        tags: Tags
-        tag_ids: Tags
+        tags: TaCategoriesgs
```
Comment (braktar) [inline config/locales/en.yml:549]:
```suggestion
        tags: Categories
```

**Review by braktar — CHANGES_REQUESTED** [review]

File: config/locales/en.yml (hunk 1 above)
Comment (frodrigo) [reply config/locales/en.yml]: Fixed, and also similar issues in other languages.

File: config/locales/en.yml (hunk 2 above)
Comment (frodrigo) [reply config/locales/en.yml]: Fixed

**Pull request merged**